code/modules/surgery/bodyparts/_bodyparts.dm

```diff
@@ -1054,7 +1054,13 @@
 
 	if(dropped && dmg_overlay_type)
 		if(brutestate)
-			. += image('icons/mob/effects/dam_mob.dmi', "[dmg_overlay_type]_[body_zone]_[brutestate]0", -DAMAGE_LAYER)
+			// divided into two overlays: one that gets colored and one that doesn't.
+			var/image/brute_blood_overlay = image('icons/mob/effects/dam_mob.dmi', "[dmg_overlay_type]_[body_zone]_[brutestate]0", -DAMAGE_LAYER)
+			brute_blood_overlay.color = get_blood_dna_color(GET_ATOM_BLOOD_DNA(src))
```

Comment:
Doesn't this mean bludgeoning someone with a detached arm will change the color of its damage overlay?